backend/config/data_providers/api_particulier.yml

```
@@ -965,5 +967,32 @@ api_particulier:
           job: Directeur R&D
           email: laurent.delattre@3douest.com
           phone_number: '0256662007'
-
-
+    coexya:
+      label: >-
+        ICAR
```

Ici je recommanderais de toujours sépcigier dans le titre quel est le cas d'usage, par exemple : 

```suggestion
        Coexya, solution portail familles ICAR
```